### DISCUSSION TITLE
How to center icons in the dropdown menu of the sidebar (DocsLayout)?

### DISCUSSION THREAD

**Discussion by nktnet1** [fuma-nama/fumadocs#823] [Q&A] [answered]
As you can see from the image below:

![image](https://github.com/user-attachments/assets/1a71904e-9c42-41a4-a502-8851c230c0fa)

The first icon for the currently selected doc is centered, but the bottom 2 options in the opened dropdown menu are not.
Is there a way to fix this?

Thank you!

---

```tsx
    <DocsLayout
      tree={source.pageTree}
      {...baseOptions}
      containerProps={{ className: 'min-h-screen' }}
      links={[
        {
          url: '/',
          text: 'Go Home',
        },
      ]}
      sidebar={{
        banner: (
          <RootToggle
            options={[
              {
                title: '24T3',
                description: 'Current',
                icon: <Omega />,
                url: '/24T3',
              },
              {
                title: '25T1',
                description: 'Next term',
                icon: <ShieldQuestion />,
                url: '/25T1',
              },
            ]}
          />
        ),
      }}
    >
      {children}
    </DocsLayout>

```

I think it's a similar issue on this page:
- https://fumadocs.vercel.app/docs/ui/components/root-toggle
    ![image](https://github.com/user-attachments/assets/1b510a0f-0526-4574-965d-1afeb4aa1f5d)

**Comment by fuma-nama** [comment, marked as answer]
See https://github.com/fuma-nama/fumadocs/discussions/758
